usaspending_api/disaster/tests/integration/test_overview.py

```diff
@@ -188,7 +188,7 @@ def test_award_obligation_adds_values(
     helpers.reset_dabs_cache()
     resp = client.get(OVERVIEW_URL)
     assert resp.data["spending"]["award_obligations"] == Decimal("2.3")
-    assert resp.data["spending"]["award_outlays"] == Decimal("1.15")
+    assert resp.data["spending"]["award_outlays"] == Decimal("1.75")
```

Comment:
Would it be too much trouble to leave the test asserts as they were and simply adjust the mock data so the sum matches the original value?